shared/persist-artefacts.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+set -e
+ARGV_DIRECTORY="$1"
+set -u
+
+mkdir -p $(pwd)/${ARGV_DIRECTORY}/.versionbot/artefacts
```

Comment:
@roman-mazur I looked this up a couple of years back and have stuck to artefact ever since https://grammarist.com/spelling/artefact-artifact/ 
My loyalty is to the queen :P